pkg/bootstrapper/templates/kubelet_config.json

```diff
@@ -0,0 +1 @@
+{"kind":"KubeletConfiguration","apiVersion":"kubelet.config.k8s.io/v1beta1","rotateCertificates":true,"serverTLSBootstrap":true,"resolvConf":"","authentication":{"x509":{"clientCAFile":"{{.ClientCAFile}} "},"anonymous":{"enabled":false}},"clusterDomain":"cluster.local","clusterDNS":["172.30.0.10"],"cgroupsPerQOS":false,"runtimeRequestTimeout":"10m0s","maxPods":250,"kubeAPIQPS":50,"kubeAPIBurst":100,"serializeImagePulls":false,"featureGates":{"LegacyNodeRoleBehavior":false,"NodeDisruptionExclusion":true,"RotateKubeletServerCertificate":true,"SCTPSupport":true,"ServiceNodeExclusion":true,"SupportPodPidsLimit":true},"containerLogMaxSize":"50Mi","systemReserved":{"cpu":"500m","ephemeral-storage":"1Gi","memory":"1Gi"},"enforceNodeAllocatable":[]}
```

Comment:
Why do we need resolvConf?

Reply:
New line.

Reply:
The default value for `resolvConf` is `/etc/resolv.conf` which is not required for Windows.

Reply:
So, we if you make it "", would it point to nothing. IIRC, in the command line flags, it shows up as `/etc/resolv.conf` if give it empty